content/03-reference/01-tools-and-interfaces/06-prisma-studio.mdx

```diff
@@ -7,7 +7,13 @@ experimental: true
 
 ## Overview
 
-Prisma Studio is a visual editor for your database. To run Prisma Studio, run the following command in your terminal:
+Prisma Studio is a very powerful database GUI where you can interact with your data.
+
+## Not Your Typical Database GUI
```

Comment:
also not too sure about the wording here, sounds a bit too marketing-y for a page in the reference docs to me. if this section is about comparing Prisma Studio to other DB GUIs, maybe something like "How Prisma Studio is different from other database GUIs" might work better?